src/shared/components/lazyMobXTable/utils.ts

```diff
@@ -0,0 +1,8 @@
+import {SHOW_ALL_PAGE_SIZE} from "../paginationControls/PaginationControls";
+export function maxPage(displayDataLength:number, itemsPerPage:number) {
```

Comment:
i think "pages" should not be zero indexed.  i.e. we don't ever have zero pages.  we have one page, two pages.

Reply:
in my experience indexing math always works out easier when things are zero-indexed